### PR TITLE
fix: center chain badge on wallet avatar

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-7251

**Info: I could not get this centered 100% correctly!**

Testing:
- Background-Color (as seen in the image underneath) is not happening on code! But rather just for testing
- to test properly, I applied the following styling temporarily to the Span-Contaner: `span: {background-color: rgba(0,0,0,0.5}; border-radius: 44px}`
- attached image is based on a 200% zoom level + browser zoom
- so going into the nitty gritty details!

image

### CODE REVIEW

**Comment by dennyscode** [comment]
While investigating on where this was coming from I came to the conclusion that some changes in this PR resulted in this issue:
https://github.com/lifinance/jumper.exchange/pull/801

File: src/components/Menus/WalletMenu/WalletMenu.style.ts [hunk 1]
```diff
@@ -78,9 +78,17 @@ export const WalletCardButtonContainer = styled(Container)(({ theme }) => ({
...
+    height: 44,
+    padding: 0,
+    transform: 'unset',
+    bottom: '-11.5px',
+    right: '-13.5px',
+  },
   '> .MuiAvatar-root': {
     overflow: 'hidden',
-    '--g': '#0000 98%,#000',
+    '--g': '#0000 100%,#000',
```
Comment (chybisov) [inline src/components/Menus/WalletMenu/WalletMenu.style.ts:91]: I think it looks better with a bit of blur when not zoomed in. Without blur, there is no antialiasing effect 
image

Comment (chybisov) [reply src/components/Menus/WalletMenu/WalletMenu.style.ts:91]: Also, I think we should have different masks for different image sizes. If you take a look at the small icon, there is a bigger cutout
image

**Review by chybisov — CHANGES_REQUESTED** [review]

**Review by tcheee — APPROVED** [review]

File: src/components/Menus/WalletMenu/WalletMenu.style.ts (hunk 1 above)
Comment (dennyscode) [reply src/components/Menus/WalletMenu/WalletMenu.style.ts:91]: Good points! Here is some updates:

WalletCard
image

WalletButton
image

Both:
image

Comment (dennyscode) [reply src/components/Menus/WalletMenu/WalletMenu.style.ts:91]: Updated again:
image

**Comment by dennyscode** [comment]
image
image

**Pull request merged**